.sanity-ansible-ignore-2.12.txt

```diff
@@ -2,3 +2,4 @@ plugins/modules/firewall_lib.py validate-modules:missing-gplv3-license
 plugins/modules/firewall_lib_facts.py validate-modules:missing-gplv3-license
 plugins/modules/firewall_lib.py validate-modules:missing-examples
 roles/firewall/files/get_files_checksums.sh shebang!skip
+files/test_ping.sh shebang!skip
```

Comment:
`tests/firewall/files/test_ping.sh`

ansible-test is only run on the converted collection - in the collection the path is `tests/firewall/files/test_ping.sh`